### PR TITLE
Delete logs

### PR DESCRIPTION
**Pull request by lbwexler**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [X] Caught up with `develop` branch as of last change.
- [X] Added CHANGELOG entry, or determined not required.
- [X] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [X] Updated doc comments / prop-types, or determined not required.
- [X] Reviewed and tested on Mobile, or determined not required.
- [X] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: admin/tabs/server/logViewer/LogViewerModel.js [hunk 1]
```diff
@@ -103,6 +112,26 @@ export class LogViewerModel extends HoistModel {
         }
     }
 
+    async deleteSelectedAsync() {
+        try {
+            const recs = this.filesGridModel.selectedRecords,
+                count = recs.length;
+            if (!count ||
+                !(await XH.confirm({title: 'Confirm Delete?', message: `Delete ${count} files?`}))) {
```
Comment (TomTirapani) [inline admin/tabs/server/logViewer/LogViewerModel.js:120]: I haven't seen this pattern of using await within the condition before - cool!

Comment (TomTirapani) [reply admin/tabs/server/logViewer/LogViewerModel.js:120]: Although I must admit I find it a bit hard to parse, given that it may or may not be async depending on the previous clause. Might be clearer to split them up into two different checks, one for the early out and one for async user confirmation?

**Comment by amcclain** [comment]
Thanks - how very convenient that this was already available as a hoist-core endpoint!

Pushed minor tweaks in 330040a97cc3b2b3ef3e1f7175300588cc85ed15 to use RecordActions and have both delete and download in the context menu + bottom toolbar.  I had been meaning to move the download option for a while now due to space constraints on the top toolbar, so this was a nice prompt to do so.
